app/uk/gov/hmrc/bindingtariffclassification/repository/CaseRepository.scala

```diff
@@ -237,6 +238,11 @@ class CaseMongoRepository @Inject() (
     )
   }
 
+  private def coalesce(fieldChoices: NonEmptySeq[String]): JsValue =
+    fieldChoices.init.foldRight(JsString(fieldChoices.last): JsValue) {
+      case (field, expr) => Json.obj("$ifNull" -> Json.arr(field, expr))
```

Comment:
```suggestion
      case (field, expr) => Json.obj("$ifNull" -> Json.arr("$" + field, expr))
```
We can add them here instead